app/forms/idv/api_image_upload_form.rb

```diff
@@ -197,10 +197,22 @@ def update_analytics(client_response)
       update_funnel(client_response)
       track_event(
         Analytics::IDV_DOC_AUTH_SUBMITTED_IMAGE_UPLOAD_VENDOR,
-        client_response.to_h,
+        client_response.to_h.merge(image_metadata),
       )
     end
 
+    def image_metadata
+      params.permit(:front_metadata, :back_metadata).
+        to_h.
+        transform_values do |str|
+          JSON.parse(str)
+        rescue JSON::ParserError
+          nil
+        end.
+        compact.
+        symbolize_keys
```

Comment:
Hm, I don't think we want to get rid of `symbolize_keys` from the return value of this method. While it is probably effectively equivalent as far as how we're logging, it results in a mix of string and symbol keys in the analytics hash:

```
           :billed=>true,
           :remaining_attempts=>4,
           :user_id=>nil,
           "front_metadata"=>
            {"width"=>40, "height"=>40, "mimeType"=>"image/png", "source"=>"upload"},
           "back_metadata"=>
            {"width"=>20,
             "height"=>20,
             "mimeType"=>"image/png",
             "source"=>"upload"}}],
```

...but! We might still want `symbolize_names` in addition to `symbolize_keys`, so that the nested keys are also symbols and not strings, though again for consistency and not having much of an effective difference.

Reply:
> ...but! We might still want `symbolize_names` in addition to `symbolize_keys`, so that the nested keys are also symbols and not strings, though again for consistency and not having much of an effective difference.

This was addressed in 99b280434b1c664856cef881f5b6d62f272a16bb.